coral-trino/src/test/java/com/linkedin/coral/trino/trino2rel/TrinoToRelConverterTest.java

```diff
@@ -0,0 +1,203 @@
+/**
+ * Copyright 2021-2022 LinkedIn Corporation. All rights reserved.
+ * Licensed under the BSD-2 Clause license.
+ * See LICENSE in the project root for license information.
+ */
+package com.linkedin.coral.trino.trino2rel;
+
+import java.util.HashMap;
+import java.util.Iterator;
+import java.util.List;
+import java.util.Map;
+
+import com.google.common.collect.ImmutableList;
+
+import org.apache.calcite.plan.RelOptUtil;
+import org.apache.calcite.rel.RelNode;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+import static org.testng.Assert.*;
+
+
+public class TrinoToRelConverterTest {
+  public static TrinoToRelConverter converter;
+
+
+  @BeforeClass
+  public void beforeClass() {
+    // TODO: migrate to use HiveMetastoreClient
+    Map<String, Map<String, List<String>>> localMetastore = new HashMap<>();
+    HashMap<String, List<String>> db = new HashMap<>();
+    localMetastore.put("default", db);
+
+    db.put("foo", ImmutableList.of("show|int", "a|int", "b|int", "x|date", "y|date"));
+    db.put("my_table", ImmutableList.of("x|array<int>", "y|array<array<int>>", "z|int"));
+    db.put("src", ImmutableList.of("x|int", "k|int", "v|string"));
+    db.put("emp", ImmutableList.of("depname|string", "empno|int", "salary|int"));
+
+    db.put("a", ImmutableList.of("b|int", "id|int", "x|int"));
+    db.put("b", ImmutableList.of("foobar|int", "id|int", "y|int"));
+
+    this.converter = new TrinoToRelConverter(localMetastore);
+  }
+
+  @DataProvider(name = "support")
+  public Iterator<Object[]> getSupportedSql() {
+    return ImmutableList.<List<String>> builder()
+        .add(ImmutableList.of("select * from foo",
+            "LogicalProject(show=[$0], a=[$1], b=[$2], x=[$3], y=[$4])\n"
+                + "  LogicalTableScan(table=[[hive, default, foo]])\n"))
+        .add(ImmutableList.of("select * from foo /* end */",
+            "LogicalProject(show=[$0], a=[$1], b=[$2], x=[$3], y=[$4])\n"
+                + "  LogicalTableScan(table=[[hive, default, foo]])\n"))
+        .add(ImmutableList.of("/* start */ select * from foo",
+            "LogicalProject(show=[$0], a=[$1], b=[$2], x=[$3], y=[$4])\n"
+                + "  LogicalTableScan(table=[[hive, default, foo]])\n"))
```

Comment:
We should probably have a SQL to SQL test (e.g., Trino to Hive/Spark along the lines of Hive to Trino). Could you summarize current HiveToRel testing flavors? We may want to follow them.